### ISSUE TITLE
puppet-openldap uses non-existent function: validate_number

### ISSUE THREAD

**Issue by Jamyn** [bodgit/puppet-openldap#38]
puppet-openldap 1.1.4 uses non-existent function **_validate_number_**:

```
manifests/configuration.pp:    validate_number($tls_protocol_min)
manifests/server.pp:    validate_number($ssl_protocol)

```

validate_number does not exist in (the latest) bodgitlib 1.4.0.

**Comment by bodgit** [comment]
This should be a stdlib function; it should be either `validate_integer` or `validate_numeric` and I've had a brain fart and typed `validate_number`. Let me take a look.

**Comment by bodgit** [comment]
So it should be `validate_numeric` as the TLS protocol numbers can be specified as `3.2` to require TLSv1.1, etc.

Fixed in master pending a v1.2.0 release. Thanks for the bug report!
